From 16f7fad2e36fdfd0579ef677d4d973b724de4f84 Mon Sep 17 00:00:00 2001
From: Henry Chen <henry.chen@oss.cipunited.com>
Date: Mon, 5 Jan 2026 17:15:40 +0800
Subject: [PATCH] FROMLIST #143432: Skip NaN encoding assertions for MIPS

---
 Lib/test/test_struct.py | 12 ++++++++----
 1 file changed, 8 insertions(+), 4 deletions(-)

diff --git a/Lib/test/test_struct.py b/Lib/test/test_struct.py
index cceecdd526c..e9f6c97a807 100644
--- a/Lib/test/test_struct.py
+++ b/Lib/test/test_struct.py
@@ -942,10 +942,14 @@ def test_half_float(self):
         else:
             expected = 0x7e
 
-        packed = struct.pack('<e', math.nan)
-        self.assertEqual(packed[1] & 0x7e, expected)
-        packed = struct.pack('<e', -math.nan)
-        self.assertEqual(packed[1] & 0x7e, expected)
+        # Skip NaN encoding checks for MIPS because `math.nan` changes its value
+        # depending on toolchain settings. See:
+        # https://gcc.gnu.org/onlinedocs/gcc/MIPS-Options.html#index-mnan_003d2008
+        if not platform.machine().startswith('mips'):
+            packed = struct.pack('<e', math.nan)
+            self.assertEqual(packed[1] & 0x7e, expected)
+            packed = struct.pack('<e', -math.nan)
+            self.assertEqual(packed[1] & 0x7e, expected)
 
         # Checks for round-to-even behavior
         format_bits_float__rounding_list = [
-- 
2.52.0

